### PR TITLE
stub out twitter pvl responses

### PR DESCRIPTION
**Pull request by pzduniak**

### CODE REVIEW

**Comment by pzduniak** [comment]
tested out by adding `127.0.0.1 mobile.twitter.com` to `/etc/hosts`

File: go/libkb/api_stub_html.go [hunk 1]
```diff
@@ -0,0 +1,577 @@
+package libkb
+
+var htmlAPIStubs = map[string]string{
```
Comment (zapu) [inline go/libkb/api_stub_html.go]: might as well try to integrate `vcr` in client codebase, what do you think @patrickxb ?

Comment (patrickxb) [reply go/libkb/api_stub_html.go]: sure, it's already used in client for stellar.

but whatever's easiest.  if this is done, then that's fine...

(also, i know we already have/had this somewhere for github and maybe twitter...maybe it was deleted)

Comment (patrickxb) [reply go/libkb/api_stub_html.go]: oh, it's this api_stub.go file.  

@zapu i say just keep using this api_stub system for now and move on to better things...

Comment (zapu) [reply go/libkb/api_stub_html.go]: sounds good. I didn't know it was used in client, I couldn't find it. I thought it was only in stellard and stellarnet

Comment (pzduniak) [reply go/libkb/api_stub_html.go]: @patrickxb merge ok? we don't have to wait for Max if you think this solution is fine.

File: go/libkb/api_stub_html.go [hunk 2]
```diff
@@ -0,0 +1,577 @@
+package libkb
```
Comment (patrickxb) [inline go/libkb/api_stub_html.go]: this file needs a comment header, and a build tag so this doesn't get into production builds (like in api_stub.go)

Comment (pzduniak) [reply go/libkb/api_stub_html.go]: corrected, wdyt now

**Pull request merged**